climate_scanner/trends_innovation_classifier/src/fasttext/modules/trends.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import os
+import yaml
+from .data_utils import load_params
+
+import glob
+import fasttext
+import operator
+from operator import itemgetter
+
+
+
+# _ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def list_all_classifiers():
+    """
+    A function which lists the available classifiers.
+    :return: list of categories of trends and innovations for which there are classifiers.
+    """
+    
+    params = load_params()
+    inputfilepath = params['data']['path_to_annotated_data']
+    categories = []
+    inputfilename = inputfilepath + "/*.jsonl"
+    filenames = glob.glob(inputfilename)
+    
+    for fn in filenames:
+        head, tail = os.path.split(fn) 
+        categories.append(tail.split('_sentences')[0])    
+    
+    return categories
+
+
+def predict_using_individual_classifier(text, model_classifier):
+    """
+    A function which predicts the output from an individual classifier.
+    :inputs: 
+        text: text to predict.
+        model_classifier: classifier to use for the prediction.
+    :return: 
+        predict_prob: prediction confidence.
+        predict_label: predicted class.
+    """
+    
+    params = load_params()
+    modelfilepath = params['data']['path_to_fasttext_individual_models'] + model_classifier + "_ft.bin"
+    predict_label, predict_prob = [], []
+    
+    model = fasttext.load_model(modelfilepath)
```

Comment:
This is something to heavily avoid, if you are loading the model into memory from disk every time you want to make a prediction, things will be very slow.